src/lib/cozy-client/slices/synchronization.js

```diff
@@ -36,14 +36,38 @@ const doctypeSync = (state = doctypeSyncInitialState, action) => {
   }
 }
 
-const synchronization = (state = {}, action) => {
+const syncInitialState = {
+  started: false,
+  initialStatus: 'pending',
+  doctypes: {}
+}
+
+const synchronization = (state = syncInitialState, action) => {
   switch (action.type) {
+    case START_SYNC:
+      return {
+        ...state,
+        started: true
+      }
+    case INITIAL_SYNC_OK:
+      return {
+        ...state,
+        initialStatus: 'ok'
```

Comment:
I think we should define internal constants for `ok` and `error`.

Reply:
It makes sense, but I find it can be harder to read in cases like this. Will do, though